scripts/cleanDB.sql

```diff
@@ -0,0 +1,23 @@
+delete from message_destinations_messages;
```

Comment:
I run it manually. It is currently complete.
It's very convenient to see what data was exactly created by a single test: clean the DB and run the test.